src/vstart.sh

```diff
@@ -542,69 +544,71 @@ ms bind msgr1 = true
         plugin dir = $CEPH_LIB
         filestore fd cache size = 32
         run dir = $CEPH_OUT_DIR
-	crash dir = $CEPH_OUT_DIR
+        crash dir = $CEPH_OUT_DIR
         enable experimental unrecoverable data corrupting features = *
-	osd_crush_chooseleaf_type = 0
-	debug asok assert abort = true
+        osd_crush_chooseleaf_type = 0
+        debug asok assert abort = true
 $msgr_conf
 $extra_conf
 EOF
-	if [ "$lockdep" -eq 1 ] ; then
-		wconf <<EOF
+    if [ "$lockdep" -eq 1 ] ; then
+        wconf <<EOF
         lockdep = true
 EOF
-	fi
-	if [ "$cephx" -eq 1 ] ; then
-		wconf <<EOF
-	auth cluster required = cephx
-	auth service required = cephx
-	auth client required = cephx
+    fi
+    if [ "$cephx" -eq 1 ] ; then
+        wconf <<EOF
+        auth cluster required = cephx
+        auth service required = cephx
+        auth client required = cephx
 EOF
-	elif [ "$gssapi_authx" -eq 1 ] ; then
-		wconf <<EOF
-	auth cluster required = gss
-	auth service required = gss
-	auth client required = gss
-	gss ktab client file = $CEPH_DEV_DIR/gss_\$name.keytab
+    elif [ "$gssapi_authx" -eq 1 ] ; then
+        wconf <<EOF
+        auth cluster required = gss
+        auth service required = gss
+        auth client required = gss
+        gss ktab client file = $CEPH_DEV_DIR/gss_\$name.keytab
 EOF
-	else 
-		wconf <<EOF
-	auth cluster required = none
-	auth service required = none
-	auth client required = none
+    else
+        wconf <<EOF
+        auth cluster required = none
+        auth service required = none
+        auth client required = none
 EOF
-	fi
-	if [ "$short" -eq 1 ]; then
-		COSDSHORT="        osd max object name len = 460
-        osd max object namespace len = 64"
-	fi
-        if [ "$objectstore" == "bluestore" ]; then
-            if [ "$spdk_enabled" -eq 1 ]; then
-                if [ "$(get_pci_selector_num)" -eq 0 ]; then
-                    echo "Not find the specified NVME device, please check." >&2
-                    exit
-                fi
-                if [ $(get_pci_selector_num) -lt $CEPH_NUM_OSD ]; then
-                    echo "OSD number ($CEPH_NUM_OSD) is greater than NVME SSD number ($(get_pci_selector_num)), please check." >&2
-                    exit
-                fi
-                BLUESTORE_OPTS="        bluestore_block_db_path = \"\"
+    fi
+    if [ "$short" -eq 1 ]; then
+        COSDSHORT="
+        osd max object name len = 460
```

Comment:
this changes the behavior of this script. and it's wrong.

Reply:
I thought that adding a new line above it was OK. Well  I'll modify it to keep the original one.